src/net_permissions.cpp

```diff
@@ -61,7 +71,16 @@ bool TryParsePermissionFlags(const std::string& str, NetPermissionFlags& output,
         readen++;
     }
 
+    // By default, whitelist only applies to incoming connections
+    if (connection_direction == ConnectionDirection::None) {
+        connection_direction = ConnectionDirection::In;
+    } else if (flags == NetPermissionFlags::None) {
```

Comment:
Going over this PR again, shouldn't this have been a separate `if` statement instead of an `if/else` (or not a condition at all)?

Right now we are accepting expressions of the type `@ip:port` (which where already accepted before), but not expressions of the type `in/out@ip:port`.

The former will pass and default to `ConnectionDirection::In +  NetPermissionFlags::Implicit`, while the latter will be rejected. That feels inconsistent